### ISSUE TITLE
Select component does not update ui selected option text after updating the options and set model programatically

### ISSUE THREAD

**Issue by HassanDomeDenea** [livewire/flux#100]
I created Volt component:

```
<div>
        <div>{{$studentId}}</div>
        <flux:select wire:model.live="studentId" variant="listbox">
            @foreach($this->students as $student)
                <flux:option value="{{$student->id}}" :key="$student->id"> {{ $student->name }} </flux:option>
            @endforeach
        </flux:select>
        <flux:input wire:model="newStudentName"/>
        <flux:button wire:click="createStudent">Create</flux:button>
    </div>

```

and php part:

```
use App\Models\Student;
use Livewire\Volt\Component;

new class extends Component {
    public $studentId = null;
    public $students = [];
    public $newStudentName = '';

    public function mount()
    {
        $this->students = Student::all();
    }

    public function createStudent()
    {
        $newStudentModel = Student::create(['name' => $this->newStudentName, 'gender' => 'male']);
        $this->students = Student::all();
        $this->studentId = $newStudentModel->id;
        dump("Done");
    }
}
```

When I click the create button, it creates the model, and update the studentId variable, but the select component selected option stay blank. When I view the options I find the last created one, but I have to select other one then select it again to update the ui.

**Comment by WorthLMS** [comment]
https://github.com/livewire/flux/issues/60 Known issue with variant="listbox"

**Comment by HassanDomeDenea** [comment]
> #60 Known issue with variant="listbox"

It is the same in combobox and default variants aslo in my situation.

**Comment by patrykszady** [comment]
Caleb on issue #60:
“ Thank you for the report. This issue has been fixed and will be available in the next release.”

Please close this once confirmed fixed in 1.0.8

**Comment by calebporzio** [comment]
This should be fixed, however, if it is not, please re-submit with an example that doesn't rely on external classes like `Student::`
